### PR TITLE
[Password reset] The “Email” input field is overlapped by description text. #69

### LINKED ISSUE #69: [Password reset] The “Email” input field is overlapped by description text.
**Description:** The “Email” input field is overlapped by description text on the password reset pop-up.

**Environment:** Windows 10 x64, Microsoft Edge 115.0.1901.203, Google Chrome 116.0.5845.96.

**Reproducible:** always.

**Build found:**

**Steps to reproduce:**
1. Click on the “Login” button on the header of the home page.
2. Click on the “Forgot password?” button on the “Password reset” pop-up.
3. Take a look at the “Email” input field.

**Actual result:**
The “Email” input field is overlapped by description text on the password reset pop-up.
Overlapped

**Expected result:**
The “Email” input field is displayed aligned on the password reset pop-up.
image

**User story and test case links**
User Story User password reset #13

### PR DESCRIPTION
**Pull request by docia**
added mb for wrapper in titleWithDescription

### CODE REVIEW

File: src/constants/translations/en/login.json [hunk 1]
```diff
@@ -15,4 +15,4 @@
   "resetPasswordDesc": "Your new password must be different from previous used passwords.",
   "savePassword": "Save password",
   "sucsessReset": "Your password has been successfully updated!"
```
Comment (Olenka-Hryk) [inline src/constants/translations/en/login.json:17]: Could you please find all instances of the variable "sucsessReset" and change them to "successReset"?

Comment (Olenka-Hryk) [inline src/constants/translations/en/login.json:17]: Could you please find all instances of the variable "sucsessReset" and change them to "successReset"?

**Review by Olenka-Hryk — APPROVED** [review]

**Review by m6xcom — APPROVED** [review]

File: src/constants/translations/en/login.json (hunk 1 above)
Comment (Nazar-Lysak) [reply src/constants/translations/en/login.json:17]: Is this a part of current issue? Maybe would be better to open another one?

**Review by Nazar-Lysak — APPROVED** [review]

File: src/constants/translations/en/login.json (hunk 1 above)
Comment (vlad-khrychov) [reply src/constants/translations/en/login.json:17]: Agree with creating a new issue, , has anyone opened it?

**Review by vlad-khrychov — APPROVED** [review]

File: src/constants/translations/en/login.json (hunk 1 above)
Comment (Nazar-Lysak) [reply src/constants/translations/en/login.json:17]: I will open a new issue on it

**Pull request merged**